qiime2_pipeline.bash

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+if [ $# -ne 4 ]; then
+	echo "Usage: $0 WORK_DIR MAPPING_FP LANE_NUM PROJECT_NAME"
+	exit 1
+fi
+
+WORK_DIR=$1
+MAPPING_FP=$2
+LANE_NUM=$3
+PROJECT_NAME=$4
+
+## GreenGene Training Set artiffact FP
+## This can be downloaded from https://data.qiime2.org/2017.8/common/gg-13-8-99-nb-classifier.qza
+gg_FP=$HOME"/gg-13-8-99-nb-classifier.qza"
```

Comment:
I recommend the following pattern:

    gg_FP="${HOME}/gg-13-8-99-nb-classifier.qza"

this applies throughout the script.

Reply:
This was reflected in the modified version. 

Thanks,
Jung-Jin